common/src/main/java/org/jboss/pnc/bacon/common/Path.java

```diff
@@ -0,0 +1,8 @@
+package org.jboss.pnc.bacon.common;
+
+public class Path {
+
+    public static String CONFIG_FOLDER = System.getProperty("user.home") + "/.config/pnc-bacon";
```

Comment:
I was little bit confused about which config is for what and I was trying to setup the Config.configLocation first, but since I need the flag parameter to be set in AbstractCommand I can't access the static part of the configuration (e.g. Config or CacheFile).